### ISSUE TITLE
Make it possible to mask filters

### ISSUE THREAD

**Issue by ghost91-** [gambit07/fxmaster#68]
In GitLab by @mesfoliesludiques on Jun 27, 2021, 16:35
https://gitlab.com/mesfoliesludiques/foundryvtt-fxmaster/-/issues/136

This could be achieved by rendering the mask to a `PIXI.RenderTexture` (e.g. via a `CachedContainer`), so that the texture can be used as a mask in the shaders of the filters.

**Comment by DanielTheBard** [comment]
Is there any hope of making this work now, say with Regions, like the particle effects do? (In particular, I've been wanting to utilize the underwater effect or something like it on bodies of liquid on maps)

**Comment by gambit07** [comment]
> Is there any hope of making this work now, say with Regions, like the particle effects do? (In particular, I've been wanting to utilize the underwater effect or something like it on bodies of liquid on maps)

Yes, this will be released in V7. The initial release will be Patreon only soon, general release likely October/November
